src/state/swap/hooks.ts

```diff
@@ -179,6 +181,11 @@ export function useDerivedSwapInfo(platformOverride?: RoutablePlatform): UseDeri
   useEffect(() => {
     const [inputCurrencyBalance] = relevantTokenBalances
 
+    // Abort previous controller and create new one
```

Comment:
this should be replaced with a function with a descriptive name and then we can remove the comment